contracts/lib/misc/ERCProxy.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.18;
+
+
+interface ERCProxy {
```

Comment:
Although ideally this would be an `interface`, I think having `FORWARDING` and `UPGRADEABLE` being in this contract is more valuable than trying to keep this as a pure `interface`.